DataProcessing/datax-host/src/main/scala/datax/input/BatchBlobInputSetting.scala

```diff
@@ -33,8 +33,8 @@ object BatchBlobInputSetting {
 
     InputBlobsConf(
       path = KeyVaultClient.resolveSecretIfAny(dict.getOrNull(SettingPath)),
-      startTime = dict.getOrNull(SettingProcessStartTime),
-      endTime = dict.getOrNull(SettingProcessEndTime),
+      startTime = dict.getOrElse(SettingProcessStartTime, sys.env.getOrElse("process_start_datetime", "")),
```

Comment:
In sys.env.getOrElse, should we prefer getOrNull instead since with that the behavior will still be the same if both SettingProcessStartTime and process_start_datetime are not specififed

Reply:
there's no getOrNull  method in sys.env